libs/blocks/caas-marquee-metadata/caas-marquee-metadata.js

```diff
@@ -14,7 +14,7 @@ function getUrl(ctaLink) {
   if (modalPath && modalHash) {
     return `${modalPath}${modalHash}`;
   }
-  return ctaLink.href;
+  return `${ctaLink.href}${ctaLink.target}`;
```

Comment:
should be just 
`return ctaLink.href;`
The target should be included on the link for the viewer top parse and decorate the link